src/main/java/org/cactoos/iterable/Mapped.java

```diff
@@ -42,7 +46,7 @@
      * @param src Source iterable
      */
     @SafeVarargs
-    public Mapped(final Func<X, Y> fnc, final X... src) {
+    public Mapped(final Func<? super X, Y> fnc, final X... src) {
```

Comment:
@andreoss it's missing the `? extend Y` in the second func type parameter.

Reply:
@victornoel Done